doc/source/user_guide/timeseries.rst

```diff
@@ -2220,6 +2220,26 @@ you can use the ``tz_convert`` method.
 
    rng_pytz.tz_convert('US/Eastern')
 
+.. note::
+
+    .. versionadded:: 1.1.0
+
+    For ambiguous times, pandas supports explicitly specifying the keyword-only fold argument.
+    Due to daylight saving time, one wall clock time can occur twice when shifting
+    from summer to winter time; fold describes whether the datetime-like corresponds
+    to the first (0) or the second time (1) the wall clock hits the ambiguous time.
+    Fold is supported only for constructing from naive ``datetime.datetime`` or :class:`Timestamp`
+    or for constructing from components (see below) with ``dateutil`` timezones as ``pytz``
+    timezones do not support fold. When localizing an ambiguous datetime, we
+    recommend to rely on :meth:`Timestamp.tz_localize` instead as it gives more control.
```

Comment:
reading this, its unclear that we *should* use fold only for dateutil and prefer tz_localize for pytz. I guess be more explict.

Reply:
Changed the phrasing to be more specific.